runners/spark/src/test/java/org/apache/beam/runners/spark/ProvidedSparkContextTest.java

```diff
@@ -39,23 +39,6 @@
  * Provided Spark Context tests.
  */
 public class ProvidedSparkContextTest {
-    private static final String[] WORDS_ARRAY = {
```

Comment:
Those should stay up here I think (all static members declarations + `getSparkContextOptions`)